src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -76,6 +89,28 @@ public function iCancelThisOrder(OrderInterface $order): void
         );
     }
 
+    /**
+     * @When /^I mark (this order) as paid$/
+     */
+    public function iMarkThisOrderAsAPaid(OrderInterface $order): void
+    {
+        $this->paymentsClient->applyTransition(
+            (string) $order->getLastPayment()->getId(),
```

Comment:
Something that we should consider is to test these actions when we will access them from the order show response